Code/Java/n-ary_tree_inorder_traversal.java

```diff
@@ -0,0 +1,143 @@
+/* N-ary Tree Inorder Traversal
+ * In N-ary Tree Or Generic Tree a Node can have N no. of children
+ * Traversal means visiting every node of a tree single time
+ * In Inorder Traversal all the children data of root Node are printed from LEFT to RIGHT Except Last child then root data is printed and then the last child  */
```

Comment:
This line is quite confusing...Can you write like First the left subtree is printed followed by the root node and lastly the right subtree.